### DISCUSSION TITLE
How to hide text when it overflows

### DISCUSSION THREAD

**Discussion by gyzerok** [emilk/egui#1385] [Q&A] [answered]
Hello! I've just started using the library and enjoying it a lot so far.

One of the questions I've got: how to implement something similar to text overflow ellipsis in web?

What I want to achieve is for text to fill up all the available width and if it is overflowing just truncate the rest and show `...`.

My current code looks like this
```rust
let title: String = "Some title".into();
let text: String = "Some very long text that surely going to overflow the available width".into();

let title_label = egui::Label::new(&title).wrap(false);
let text_label = egui::Label::new(&text).wrap(false);

ui.add(title_label);
ui.add(text_label);
```

**Comment by emilk** [comment, marked as answer]
This is being worked on in https://github.com/emilk/egui/pull/1291

**Reply by elliottslaughter** [reply]
Was this merged? I'm looking for `TextWrapping` in the docs and can't find it (or `overflow`).
